### PR TITLE
Polymer 3 preview 2 blog post

### PR DESCRIPTION
**Pull request by katejeffreys**
Staged here https://polymer-3-preview-2-dot-polymer-project.appspot.com/blog/2018-01-10-polymer-3-new-preview.html

### CODE REVIEW

**Comment by katejeffreys** [comment]
A few loose ends:

>For people interested in using lit-html, an experimental base class using 
>`PropertiesMixin` with lit-html templating is available somewhere.

Do we have a link I can add?

I tweaked a couple of lines to respond to the comments in the draft - @graynorton @justinfagnani pls have a look and make sure you're happy with it :) 

Let me know any changes you need!

File: app/blog/2018-01-10-polymer-3-new-preview.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    ...
+  }
+  customElements.define('my-app-element', MyAppElement);
+  ```
+
+These new features will also be released in the next minor release of 2.x. However, there's no need to manually convert your 2.x code to use the `html` helper—the Polymer Modulizer will make that conversion for you.
+
+Other than these changes, APIs haven't changed, and you can follow the instructions in our previous blog post, [Hands-on with the Polymer 3.0 preview](2017-08-23-hands-on-30-preview).
+
+So give the latest Polymer 3.0 preview a try. Send us feedback, report bugs, and let us know what you think!
```
Comment (graynorton) [inline app/blog/2018-01-10-polymer-3-new-preview.md]: ## What's coming next?

For an idea of what's coming down the pike, keep an eye on the roadmap and element status info we've added to the `polymer-modulizer` README.

As we said above, you can expect regular preview releases for the duration of the Polymer 3.0 project—look for the next one to land the week of January 22.

Based on feedback from early adopters of 3.0, we'll be focusing next on smoothing out some rough edges around package installation and module import specifiers. Watch this space for details!

File: app/blog/2018-01-10-polymer-3-new-preview.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      return html`<div>I'm a template</div>
+                  <div>[[withBindings]]</div>
+                  <button on-click="clickHandler">Click me!</button>`
+    }
+    ...
+  }
+  customElements.define('my-app-element', MyAppElement);
+  ```
+
+These new features will also be released in the next minor release of 2.x. However, there's no need to manually convert your 2.x code to use the `html` helper—the Polymer Modulizer will make that conversion for you.
```
Comment (graynorton) [inline app/blog/2018-01-10-polymer-3-new-preview.md]: In the interest of forward compatibility, these new features have been implemented in the Polymer 2 codebase and will also be included in the next 2.x release. As with any new features, keep in mind that they're subject to change until they land in an official release.

By the way, though you'll be able to use the `html` helper in your 2.x code if you want, you don't need to manually convert all of your string-based templates–Polymer Modulizer will do that for you when you convert to 3.0.

**Comment by arthurevans** [comment]
Updated PR and staging.

https://polymer-3-preview-2-dot-polymer-project.appspot.com/blog/2018-01-10-polymer-3-new-preview.html

Depends on this roadmap PR: https://github.com/Polymer/project/pull/42/files

@graynorton @justinfagnani WDYT?

**Comment by arthurevans** [comment]
Ooops, correction, new version is staged to: https://polymer-3-preview-2-blog-dot-polymer-project.appspot.com/blog/2018-01-10-polymer-3-new-preview

(slightly different URL).

**Pull request merged**